tests/unit/dialog/dialog_core.js

```diff
@@ -59,90 +59,94 @@ asyncTest( "focus tabbable", function() {
 
 		element = $( markup ).dialog( options );
 		setTimeout(function() {
-			testFn();
-			element.remove();
-			setTimeout( next );
+			testFn(function done() {
```

Comment:
Just wondering: Is the `done` function name necessary? Or just a help for debugging?